api/net/Areas/Admin/Controllers/ReportController.cs

```diff
@@ -332,12 +332,16 @@ public IActionResult GetDashboardReport(int id)
     {
         var report = _reportService.GetDashboardReport(id);
 
-        var subscribers = report.SubscribersManyToMany.Where(s => s.User!.AccountType != Entities.UserAccountType.Distribution).Select(s => new UserReportModel(s)).ToList();
+        var subscribers = report.SubscribersManyToMany.Where(s => s.User!.AccountType != Entities.UserAccountType.Distribution && s.User!.IsVacationMode != true).Select(s => new UserReportModel(s)).ToList();
```

Comment:
Why are filtering on vacation mode here?  This endpoint returns all users who are subscribed so that we can display the status.

Reply:
Fixed.